.gitmodules

```diff
@@ -1,18 +1,21 @@
 [submodule "main/openvpn"]
 	path = main/src/main/cpp/openvpn
-	url = https://github.com/schwabe/openvpn.git
+	url = ../../schwabe/openvpn.git
 [submodule "main/openssl"]
 	path = main/src/main/cpp/openssl
-	url = https://github.com/schwabe/platform_external_openssl.git
+	url = ../../schwabe/platform_external_openssl.git
 [submodule "main/breakpad"]
 	path = main/breakpad
-	url = https://github.com/schwabe/breakpad.git
+	url = ../../schwabe/breakpad.git
 [submodule "main/src/main/cpp/mbedtls"]
 	path = main/src/main/cpp/mbedtls
-	url = https://github.com/schwabe/polarssl.git
+	url = ../../schwabe/polarssl.git
 [submodule "main/src/main/cpp/openvpn3"]
 	path = main/src/main/cpp/openvpn3
-	url = https://github.com/schwabe/openvpn3.git
+	url = ../../schwabe/openvpn3.git
 [submodule "main/src/main/cpp/asio"]
 	path = main/src/main/cpp/asio
-	url = https://github.com/chriskohlhoff/asio/
+	url = ../../chriskohlhoff/asio/
+[submodule "main/src/main/cpp/lz4"]
+	path = main/src/main/cpp/lz4
+	url = ../../lz4/lz4.git
```

Comment:
Change this to a GitHub link

Reply:
Actually, that's what I did first, but I think this actually works! because they are relative links, so whether you use `git@github.com` or `https://github.com` doesn't matter... Or are you sure this doesn't work?

Reply:
It only works for git@github.com, not with HTTPS.
So this means that if you want to build the project from CI, it needs a GitHub account to check out the project. 
But if you use HTTPS links, it can check them out without authenticating, because all of them are public repos.
See https://github.com/schwabe/ics-openvpn/issues/442#issuecomment-179905715 for more info.